spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc

```diff
@@ -0,0 +1,212 @@
+[[dashboard]]
+= Dashboard
+
+[partintro]
+--
+This section describe how to use the Dashboard of Spring Cloud Data Flow.
+--
+
+[[dashboard-introduction]]
+== Introduction
+
+Spring Cloud Data Flow provides a browser-based GUI which currently has 6 sections:
+
+* **Apps** Lists all available applications and provides the control to register/unregister them
+* **Runtime** Provides the Dataflow cluster view with the list of all running applications
+* **Streams** Deploy/undeploy Stream Definitions
+* **Tasks** List, create, launch and destroy Task Definitions
+* **Jobs** Perform Batch Job related tasks
+* **Analytics** Create data visualizations for the various analytics modules
+
+Upon starting Spring Cloud Data Flow, the Dashboard is available at:
+
+"http://<adminHost>:<adminPort>/admin-ui"
+
+For example: http://localhost:9393/admin-ui
```

Comment:
In the http single view of the document it only shows "localhost:9393/admin-ui"  probably need escape this section so that it will show http://.